server/index.js

```diff
@@ -68,6 +68,86 @@ app.get('/api/products/:productId', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/cart', (req, res, next) => {
+  const sql = `
+    select *
+    from "carts"`;
+  db.query(sql)
+    .then(result => {
+      const carts = result.rows;
```

Comment:
Just for clarity, these should be `cartItems`, not `carts`.

Reply:
Just the const needs to be cartItems, correct? the "from "carts"`" is right?